express/parsers/apps/nwchem/parser.py

```diff
@@ -0,0 +1,51 @@
+import os
+import numpy as np
+
+from express.parsers import BaseParser
+from express.parsers.apps.nwchem import settings
+from express.parsers.mixins.ionic import IonicDataMixin
+from express.parsers.mixins.reciprocal import ReciprocalDataMixin
+from express.parsers.mixins.electronic import ElectronicDataMixin
+from express.parsers.utils import find_file
+from express.parsers.apps.nwchem.formats.txt import NwchemTXTParser
+from express.parsers.utils import find_file, find_fines_by_name_substring
+
+
+class NwchemParser(BaseParser, IonicDataMixin, ElectronicDataMixin, ReciprocalDataMixin):
+    """
+    Nwchem parser class.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(NwchemParser, self).__init__(*args, **kwargs)
+        self.work_dir = self.kwargs["work_dir"]
+        self.stdout_file = self.kwargs["stdout_file"]
+        self.txt_parser = NwchemTXTParser(self.work_dir)
+
+    def total_energy(self):
+        """
+        Returns total energy.
+
+        Reference:
+            func: express.parsers.mixins.electronic.ElectronicDataMixin.total_energy
+        """
+        total_dft_energy = 27.2114 * self.txt_parser.total_energy(self._get_file_content(self.stdout_file))
```

Comment:
We should convert the units in a single place. Perhaps the txt parser can extract the value in the native units (i.e. hartree), and here we can convert them to adhere to ESSE defaults?

Reply:
I think that sounds good. I will do that.